### PR TITLE
feat(multi-stream): Add desktop sharing testcases for multi-stream mode.

### PR DESCRIPTION
**Pull request by jallamsetty1**

### CODE REVIEW

File: src/test/java/org/jitsi/meet/test/DesktopSharingTest.java [hunk 1]
```diff
@@ -46,14 +56,42 @@ public void testDesktopSharingStart()
     {
         JitsiMeetUrl url = getJitsiMeetUrl()
             .appendConfig("config.filmstrip.disableStageFilmstrip=true")
-            .appendConfig("config.flags.sourceNameSignaling=false")
-            .appendConfig("config.flags.sendMultipleVideoStreams=false");
-        ensureOneParticipant(url);
+            .appendConfig("config.p2p.enabled=true");
 
+        ensureOneParticipant(url);
```
Comment (damencho) [inline src/test/java/org/jitsi/meet/test/DesktopSharingTest.java:61]: You should not need this if you have ensureTwoParticipants after that. Not that having it causes any issues :)

**Review by damencho — APPROVED** [review]
This PR failure we see it again
```
10:57:02 FAIL: Unexpected SRTP failures in node SrtcpDecryptNode.
10:57:02 FAIL: Too many discarded packets from node SrtcpDecryptNode: 1
```
@bgrozev @JonathanLennox if its expected can we bump the threshold :)

File: src/test/java/org/jitsi/meet/test/DesktopSharingTest.java (hunk 1 above)
Comment (jallamsetty1) [reply src/test/java/org/jitsi/meet/test/DesktopSharingTest.java:61]: Hmm I don't get it. I am hanging up all the participants and then calling ensureOneParticipant to add the first participant. I am toggling screenshare on/off before adding the other two.

Comment (damencho) [reply src/test/java/org/jitsi/meet/test/DesktopSharingTest.java:61]: No, I mean ... look what ensureTwoParticipants do: https://github.com/jitsi/jitsi-meet-torture/blob/92b67127158b917d3a939bc9230666652fcf5252/src/test/java/org/jitsi/meet/test/web/WebTestBase.java#L151
It calls ensureOneParticipant :) What I mean is that there is no point of having that call of ensureOneParticipant :)

**Review by damencho — APPROVED** [review]

**Pull request merged**